modules/serverless-rn-1-28-0.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-28-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.28
+
+{ServerlessProductName} 1.28 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-28-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.7.
+* {ServerlessProductName} now uses Knative Eventing 1.7.
+* {ServerlessProductName} now uses Kourier 1.7.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.7.
+* {ServerlessProductName} now uses Knative Kafka 1.7.
+* The `kn func` CLI plug-in now uses `func` FIXME version.
+
+* Node.js and TypeScript runtimes for {ServerlessProductName} Functions are now Generally Available (GA).
+
```

Comment:
I think we could mention that Python is now available as Tech Preview for Functions. @nainaz wdyt?